adminpages/reports/sales.php

```diff
@@ -924,6 +924,43 @@ function pmpro_report_title_sales() {
 
 	</form>
 	<?php
+	// Show a table with all of the raw data.
+	?>
+	<div class="pmpro_table_area">
+		<table class="widefat striped">
+			<thead>
+				<tr>
+					<th><?php esc_html_e( 'Date', 'paid-memberships-pro' ); ?></th>
+					<th><?php esc_html_e( 'Total', 'paid-memberships-pro' ); ?></th>
+					<th><?php esc_html_e( 'New', 'paid-memberships-pro' ); ?></th>
+					<th><?php esc_html_e( 'Renewals', 'paid-memberships-pro' ); ?></th>
+				</tr>
+			</thead>
+			<tbody>
+				<?php
+				foreach ( $csvdata as $row ) {
+					// round to two decimals.
+					?>
+						<th scope="row"><?php echo date_i18n( $tooltip_date_format, strtotime( $row->date ) ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></th>
```

Comment:
If I set chart to "annual", this line is showing me the same "year" in all rows of the table. Seems that when set to "annual" $row->date is an integer. All other report date ranges are setting $row->date to a string.

Reply:
Fixed, good catch. Thank you!